esp/src/eclwatch/nls/bs/hpcc.js

```diff
@@ -105,7 +105,7 @@
     DenyRead: "<center>Zabranite<br>Čitanje</center>",
     DenyWrite: "<center>Zabranite<br>Pisanje</center>",
     Depth: "Dubina",
-    DepthTooltip: "'Najveća Dubina Podgrafa",
+    DepthTooltip: "&lsquo;Najveća Dubina Podgrafa",
```

Comment:
I created a PR to fix it - see https://github.com/hpcc-systems/HPCC-Platform/pull/9426